examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,532 @@
+{
```

Comment:
<blockquote>are modifies</blockquote><p>grammar</p><p> </p><blockquote>Therefore, let's define a placeholder <code>data_acquired</code> as a list containing 3 lists with <code>None</code> elements, which will be overwritted by <code>acquire_data_async()</code></blockquote><p>I don't get this. </p><p> </p><p>You never return the values from the functions. So , probably you should just write directly into <code>data_acquired</code>? Why are you overwriting the args?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>